spring/src/main/java/io/seata/spring/annotation/GlobalTransactionScanner.java

```diff
@@ -304,6 +313,24 @@ public void setApplicationContext(ApplicationContext applicationContext) throws
         this.setBeanFactory(applicationContext);
     }
 
+    @Override
+    public Object postProcessBeforeInitialization(Object bean, String beanName) {
+        if (!initTmClient && !disableGlobalTransaction) {
+            try {
+                Class<?> serviceInterface = SpringProxyUtils.findTargetClass(bean);
+                if (existsAnnotation(new Class[]{serviceInterface})) {
+                    initTmClient = true;
+                    if (LOGGER.isDebugEnabled()) {
+                        LOGGER.debug("Need to init tm client.");
+                    }
+                }
+            } catch (Exception e) {
+                throw new RuntimeException(e);
```

Comment:
Should any exceptions stop the starting of application?

Reply:
> Should any exceptions stop the starting of application?

findTargetClass method

Reply:
> > Should any exceptions stop the starting of application?
> 
> findTargetClass method

I think any exception here should be ignored. We can display a log to the user and tell them that TM client did not start for some reasons. However, we can not directly prevent the application from starting.What do you think?

Reply:
> > > Should any exceptions stop the starting of application?
> > 
> > 
> > findTargetClass method
> 
> I think any exception here should be ignored. We can display a log to the user and tell them that TM client did not start for some reasons. However, we can not directly prevent the application from starting.What do you think?

I think ok，
My previous consideration was that if TM did not init, but GlobalTransactional was useful in the service.
I wil fix it, thank you brother.

Reply:
> > > > Should any exceptions stop the starting of application?
> > > 
> > > 
> > > findTargetClass method
> > 
> > 
> > I think any exception here should be ignored. We can display a log to the user and tell them that TM client did not start for some reasons. However, we can not directly prevent the application from starting.What do you think?
> 
> I think ok，
> My previous consideration was that if TM did not init, but GlobalTransactional was useful in the service.
> I wil fix it, thank you brother.

please review again.